doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -1060,8 +1060,8 @@ context availability management is about the *source of the entities and attribu
 in context availability management is the *registration* resource. A registration contains information about a source of
 information (named a "context provider") and whose entities and attributes are provided by that source.
 
-Let's illustrate with an example, creating a simple registration. We are stating that the attributes temperature and
-pressure of Room5 are provided by a context provider in the URL http://mysensors.com/Rooms:
+Let's illustrate with an example, with forwards in NGSIv1 format (legacyForwarding set to true), creating a simple registration.
```

Comment:
I missed walkthrough_apiv2.md in my original analyis... It should be modified to be fully based in NGSIv2 (before this PR we cannot, but now it's possible).

Reply:
Same comment here, better to postpone until NGSIv2 forwarding is ready.

Reply:
Same https://github.com/telefonicaid/fiware-orion/pull/3467#discussion_r271797488

Reply:
Complete proof-read and corrections in 45e7eedd0.
There wasn't really much about NGSIv1